### PR TITLE
🇲🇦 Add NgMorocco Logo

### PR DESCRIPTION
**Pull request by ouadie-lahdioui**
Hi guys 👋

First of all, thank you so very this awesome project!

The aim of this PR is to add the logo of The Moroccan Angular Developer Community.

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -128,6 +128,12 @@
       "description": "Logo of the Dutch Angular Group meetup",
       "license": "MIT",
       "name": "Dutch Angular Group"
+    },
+    "ng-morocco.svg": {
+      "creator": "Ouadie LAHDIOUI",
+      "description": "Logo Of The Moroccan Angular Developer Community",
```
Comment (fmalcher) [inline metadata.json]:
```suggestion
      "description": "Logo of The Moroccan Angular Developer Community",
```

File: metadata.json [hunk 2]
```diff
@@ -128,6 +128,12 @@
       "description": "Logo of the Dutch Angular Group meetup",
       "license": "MIT",
       "name": "Dutch Angular Group"
+    },
+    "ng-morocco.svg": {
+      "creator": "Ouadie LAHDIOUI",
```
Comment (fmalcher) [inline metadata.json]:
```suggestion
      "creator": "Ouadie Lahdioui",
```

**Comment by fmalcher** [comment]
Thank you! I suggested two changes, but LGTM in general.

**Comment by ouadie-lahdioui** [comment]
@fmalcher well done! Thank you

**Comment by maartentibau** [comment]
Thanks mate!! 🙏

**Pull request merged**